sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -303,7 +316,32 @@ final class DataFrameWriter private[sql](df: DataFrame) {
       conn.close()
     }
 
-    JdbcUtils.saveTable(df, url, table, props)
+    JdbcUtils.saveTable(df, url, table, props, columnMapping)
+  }
+
+  /**
+   * (java-specific) version of jdbc method
```

Comment:
Indent is off by a space; if possible write a scaladoc link to the method above.